action/rlp_tx_test.go

```diff
@@ -299,21 +299,21 @@ var (
 			"04dc4c548c3a478278a6a09ffa8b5c4b384368e49654b35a6961ee8288fc889cdc39e9f8194e41abdbfac248ef9dc3f37b131a36ee2c052d974c21c1d2cd56730b",
 			"1e14d5373e1af9cc77f0032ad2cd0fba8be5ea2e",
 		},
-		//{
-		//	"rewardingClaim",
-		//	"f8c6806482520894a576c141e5659137ddda4223d209d4744b2106be80b8642df163ef0000000000000000000000000000000000000000000000000000000000000065000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000008224c6a03d62943431b8ca0e8ea0a9c79dc8f64df14c965ca5486da124804013778ed566a065956b185116b65cec0ec9bcf9539e924784a4b448190b0aa9d017f1719be921",
-		//	0,
-		//	21000,
-		//	"100",
-		//	"0",
-		//	"0xA576C141e5659137ddDa4223d209d4744b2106BE",
-		//	_evmNetworkID,
-		//	iotextypes.Encoding_ETHEREUM_EIP155,
-		//	100,
-		//	"4d26d0736ebd9e69bd5994f3730b05a2d48c810b3bb54818be65d02004cf4ff4",
-		//	"04830579b50e01602c2015c24e72fbc48bca1cca1e601b119ca73abe2e0b5bd61fcb7874567e091030d6b644f927445d80e00b3f9ca0c566c21c30615e94c343da",
-		//	"8d38efe45794d7fceea10b2262c23c12245959db",
-		//},
+		{
+			"rewardingClaim",
+			"f8c6806482520894a576c141e5659137ddda4223d209d4744b2106be80b8642df163ef0000000000000000000000000000000000000000000000000000000000000065000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000008224c6a03d62943431b8ca0e8ea0a9c79dc8f64df14c965ca5486da124804013778ed566a065956b185116b65cec0ec9bcf9539e924784a4b448190b0aa9d017f1719be921",
+			0,
+			21000,
+			"100",
+			"0",
+			"0xA576C141e5659137ddDa4223d209d4744b2106BE",
+			_evmNetworkID,
+			iotextypes.Encoding_ETHEREUM_EIP155,
+			100,
+			"4d26d0736ebd9e69bd5994f3730b05a2d48c810b3bb54818be65d02004cf4ff4",
+			"04830579b50e01602c2015c24e72fbc48bca1cca1e601b119ca73abe2e0b5bd61fcb7874567e091030d6b644f927445d80e00b3f9ca0c566c21c30615e94c343da",
+			"8d38efe45794d7fceea10b2262c23c12245959db",
+		},
```

Comment:
this is test with v1 ABI, fixed and restored
the original #4277 shouldn't have broken this